submissions/models.py

```diff
@@ -46,6 +43,12 @@ def __init__(self, *args, **kwargs):
         kwargs['db_index'] = True
         super(AnonymizedUserIDField, self).__init__(*args, **kwargs)
 
+    def deconstruct(self):
+        name, path, args, kwargs = super(AnonymizedUserIDField, self).deconstruct()
+        del kwargs["max_length"]
+        del kwargs["db_index"]
+        return name, path, args, kwargs
+
```

Comment:
this was reviewed here https://github.com/edx/edx-submissions/pull/26#discussion_r43888634